src/test/resources/product-json-schema.json

```diff
@@ -73,7 +73,6 @@
     },
 
     "Value": {
-      "type": "object",
```

Comment:
This is an incorrect schema example. If your schema is defined by references, it cannot be at the same time defined by a direct type. We are going to have a very hard time to explain all these details... eventually we are going to see ourselves forced to used something less broken than json schema draft v4.